### ISSUE TITLE
CSF cytology positive

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11538]
**Preferred term label:**
CSF cytology positive

**Synonyms**
cerebrospinal fluid cancer cells
cerebrospinal fluid neoplastic cells

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0476506 but this seems more like a parent term

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Carcinomatous meningitis

**Your nano-attribution (ORCID)**

**Comment by grahamponting** [comment]
Can  I suggest adding in malignancy to the label, to distinguish and make specific.
Such as **"CSF cytology positive for malignant cells"**
Just a thought.

**Comment by MickeySegal** [comment]
Adding "for malignant cells" is a good idea.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001477
- label: Malignant cells in CSF
- parents: [Abnormal cerebrospinal fluid morphology (HP:0002921)](https://hpo.jax.org/app/browse/term/HP:0002921)
- pmid: PMID:573381|PMID:27759807
- def: The identification of malignant cells in cerebrospinal fluid (CSF), generally by microscopic examination of the CSF.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11538
- synonyms: Cerebrospinal fluid neoplastic cells(NONE)|Cerebrospinal fluid cancer cells(NONE)|Positive CSF cytology(NONE)
